apis_core/collections/models.py

```diff
@@ -18,6 +33,22 @@ class SkosCollection(GenericModel, models.Model):
 
     class Meta:
         ordering = ["name"]
+        constraints = [
+            models.UniqueConstraint(
+                fields=(
+                    "name",
+                    "parent",
+                ),
+                name="unique_name_parent",
+                nulls_distinct=False,
+                violation_error_message="The name and the parent must be unique",
```

Comment:
```suggestion
                violation_error_message=\"The combination of name and base collection (parent) must be unique.\",
```
I think the phrasing of the violation message should be a bit clearer.

Reply:
I've now used `parent collection` instead of `base collection (parent)` - the term `base collection` is not in use anywhere